[FSDP][Easy] ufmt files

awgu opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#90548 [FSDP][Easy] ufmt files**
* #90546 [FSDP] Fix `use_orig_params=True` + `no_sync()`

```
ufmt format torch/distributed/fsdp
ufmt format test/distributed/fsdp
```

Commit:
[FSDP][Easy] ufmt files

[ghstack-poisoned]

Commit:
Update on "[FSDP][Easy] ufmt files"


```
ufmt format torch/distributed/fsdp
ufmt format test/distributed/fsdp
```

[ghstack-poisoned]

Commit:
Update on "[FSDP][Easy] ufmt files"


```
ufmt format torch/distributed/fsdp
ufmt format test/distributed/fsdp
```

[ghstack-poisoned]

rohan-varma reviewed (approved):
Wondering what causes the files to go out of format every now and then?

Is it possible to add pre-commit hook to ufmt FSDP or distributed related files, and also have an automatic import sorter that is also linter-checked, so that more of this can be automated?

Commit:
Update on "[FSDP][Easy] ufmt files"


```
ufmt format torch/distributed/fsdp
ufmt format test/distributed/fsdp
```

[ghstack-poisoned]

awgu commented:
> Wondering what causes the files to go out of format every now and then?
> 
> Is it possible to add pre-commit hook to ufmt FSDP or distributed related files, and also have an automatic import sorter that is also linter-checked, so that more of this can be automated?

The issue is that developers need to format it themselves. My understanding is that currently, developers must choose to set up the pre-commit hook themselves.

I think Charlie and Chien-Chin have ufmt setup since I notice their PRs will also format following ufmt.

You can download ufmt from: https://pypi.org/project/ufmt/
It is the recommended formatter for the PyTorch repo, per an internal post and also as seen in `lintrunner`:
https://github.com/pytorch/pytorch/blob/96a36c9a3b57ffca10ac4fa95d2a21f85cba78b6/.lintrunner.toml#L774-L776

awgu commented:
@pytorchbot merge
